tutorials/60-minute-blitz.jl

```diff
@@ -171,20 +169,16 @@ gradient(myloss, W, b, x)
 # indicate that we want their derivatives. `W` and `b` represent the weight and
 # bias respectively.
 
-using Flux.Tracker: param, back!, grad
-
-W = param(randn(3, 5))
-b = param(zeros(3))
+W = randn(3, 5)
+b = zeros(3)
 x = rand(5)
 
-y = sum(W * x .+ b)
+y(x) = sum(W * x .+ b)
 
-# Anything marked `param` becomes *tracked*, indicating that Flux keeping an eye
-# on its gradient. We can now call
+grads = gradient(()->y(x), params(W, b))
```

Comment:
```suggestion
grads = gradient(()->y(x), Params([W, b]))
```
and add relative import or just `using Flux`

Reply:
changed in new commit